### ISSUE TITLE
Potential color contrast and keyboard navigation issues

### ISSUE THREAD

**Issue by katemoser** [katemoser/one-thing#40]
# This is the issue description for the file /components/AllComplete.tsx

 ### Accessibility Concerns:

 - The image tag in line 10 is missing an alt attribute. This is a problem because screen readers rely on the alt attribute to describe the content of the image to visually impaired users. To fix this issue, add an alt attribute to the image tag.
 yay!
   // const options = {
  //   method: "POST",
  //   headers: {
  //     Authorization: Beare
  //     "X-GitHub-Token": gitHubToken,
  //     "Content-Type": "application/json",
  //   },
  //   body: JSON.stringify(query),
  // };
  // console.log("OPTIONS", options);

  // const response = await fe
  // console.log(response);
  // const data = await response.json();
  // console.log("ISSUE DATA:", data);
  // return data.message
